stratum/hal/lib/tdi/tdi_table_manager.cc

```diff
@@ -500,8 +500,7 @@ ::util::StatusOr<::p4::v1::TableEntry> TdiTableManager::BuildP4TableEntry(
       param->set_param_id(expected_param.id());
       param->set_value(value);
     }
-  } 
-  else {
+  }else {
```

Comment:
commit by mistake? no change to logic here except deleting a line

Reply:
no change in logic , reverted it back to original format